components/script/fetch.rs

```diff
@@ -296,6 +330,10 @@ impl FetchResponseListener for FetchContext {
         _: RequestId,
         fetch_metadata: Result<FetchMetadata, NetworkError>,
     ) {
+        // Step 12.1. If locallyAborted is true, then abort these steps.
+        if self.locally_aborted {
+            return;
+        }
```

Comment:
Step 12.2 can't be implemented yet, as `Response` doesn't have an `is_aborted` flag. It also doesn't have the underlying `Response` from the net crate